plugin/trino-faker/src/test/java/io/trino/plugin/faker/TestFakerQueries.java

```diff
@@ -42,26 +43,23 @@ void testShowTables()
     @Test
     void testColumnComment()
     {
-        assertUpdate("CREATE TABLE faker.default.comments (id INTEGER, name VARCHAR)");
-        assertUpdate("COMMENT ON COLUMN comments.name IS 'comment text'");
-
-        assertTableColumnNames("faker.default.comments", "id", "name");
-
-        assertUpdate("DROP TABLE faker.default.comments");
+        try (TestTable table = new TestTable(getQueryRunner()::execute, "cannot_comment", "(id INTEGER, name VARCHAR)")) {
+            assertUpdate("COMMENT ON COLUMN %s.name IS 'comment text'".formatted(table.getName()));
+            assertTableColumnNames("faker.default." + table.getName(), "id", "name");
+        }
     }
 
     @Test
     void testCannotCommentRowId()
     {
-        assertUpdate("CREATE TABLE faker.default.cannot_comment (id INTEGER, name VARCHAR)");
-        @Language("SQL")
-        String testQuery = "COMMENT ON COLUMN \"cannot_comment\".\"$row_id\" IS 'comment text'";
-
-        assertThatThrownBy(() -> getQueryRunner().execute(testQuery))
-                .as("Query: " + testQuery)
-                .hasMessageContaining("Cannot set comment for $row_id column");
-
-        assertUpdate("DROP TABLE faker.default.cannot_comment");
+        try (TestTable table = new TestTable(getQueryRunner()::execute, "cannot_comment", "(id INTEGER, name VARCHAR)")) {
+            @Language("SQL")
+            String testQuery = "COMMENT ON COLUMN \"%s\".\"$row_id\" IS 'comment text'".formatted(table.getName());
+
+            assertThatThrownBy(() -> getQueryRunner().execute(testQuery))
+                    .as("Query: " + testQuery)
+                    .hasMessageContaining("Cannot set comment for $row_id column");
```

Comment:
Use `assertThat(query())` instead and inline `testQuery`. (No need to use it in `as()`)